libvast/vast/detail/process.hpp

```diff
@@ -0,0 +1,34 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#pragma once
+
+#include "vast/filesystem.hpp"
+#include <caf/optional.hpp>
+
+namespace vast::detail {
+
+/// Get the path of a shared library.
```

Comment:
We usually use third person singular to describe what a function does. Here, `Finds` or `Locates` seems more appropriate to describe what this function does.

Also, a complete documentation includes a description, contains Doxygen directives for all arguments, and describes the return value. Here, `addr` is not covered.

Reply:
That comment is outdated, I will replace it with a correct one.